VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -124,6 +125,12 @@ private void bindViewHolderToThreadRow(DiscussionThreadViewHolder holder) {
 
         holder.threadBodyTextView.setText(DiscussionTextUtils.parseHtml(discussionThread.getRenderedBody()));
 
+        holder.threadVisibilityTextView.setText(discussionThread.getGroupName() == null ?
+                context.getString(R.string.discussion_post_visibility_everyone) :
```

Comment:
`getString()` can be replaced with `getText()` here, since that is more versatile, and only `CharSequence is required here.